scripts/generate_base_coverage.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import json
+import magic
+import os
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('base_dir', type=str)
+    parser.add_argument('--output', '-o', type=str, default='.')
+    args = parser.parse_args()
+
+    python_filepaths = list_python_filepaths(os.path.abspath(args.base_dir))
+    coverage_lines = {}
+    for python_filepath in python_filepaths:
+        coverage_lines[python_filepath] = []
+    coverage_dict = {"lines": coverage_lines}
```

Comment:
Is there any document about `.coverage` format? If so, could you add a comment with the link?

Reply:
I read `.coverage` file generated and learn the syntax.
I searched on google, but the syntax is not written on the website...
https://coverage.readthedocs.io/en/7.2.7/cmd.html#data-file

Reply:
instead, I wrote syntax information in the file.
[`f924495` (#6)](https://github.com/GITAI/code_coverage/pull/6/commits/f92449535c99b3be10ac03319b171ee5d745ce25)

Reply:
LGTM